android/src/main/res/values/strings.xml

```diff
@@ -115,5 +115,15 @@
     <string name="tailnet_lock_key_explainer">Used to authorize changes to the Tailnet lock configuration.</string>
     <string name="bug_report_id">Bug Report ID</string>
     <string name="learn_more">Learn more…</string>
+    <string name="dns_settings">DNS Settings</string>
+    <string name="using_tailscale_dns">Using Tailscale DNS</string>
+    <string name="this_device_is_using_tailscale_to_resolve_dns_names">This device is using Tailscale to resolve DNS names.</string>
```

Comment:
this seems to be a duplicate of the entry on line 125 - do we need two separate strings?

Reply:
nice catch!